igvc/src/flag_detector/flagdetector.h

```diff
@@ -1,6 +1,5 @@
 #ifndef FLAGDETECTOR_H
 #define FLAGDETECTOR_H
-#include <camera_info_manager/camera_info_manager.h>
```

Comment:
why?

Reply:
Camera_info_manager was ruled out

Reply:
It is linked in the CMakeLists.txt but not included as a dependency and therefore not downloaded when CI runs.

Reply:
Yes, but can we remove the link in CMakeLists.txt? I don't think it's necessary

Reply:
what does this package include?

Reply:
camera_info_manager: "This package provides a C++ interface for camera calibration information. It provides CameraInfo, and handles SetCameraInfo service requests, saving and restoring the camera calibration data."